chaoscenter/authentication/pkg/authConfig/repository.go

```diff
@@ -0,0 +1,61 @@
+package authConfig
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type Repository interface {
+	CreateConfig(config AuthConfig) error
+	GetConfig(key string) (*AuthConfig, error)
+	UpdateConfig(ctx context.Context, key string, value interface{}) error
+}
+
+type repository struct {
+	Collection *mongo.Collection
+}
+
+func (r repository) CreateConfig(config AuthConfig) error {
```

Comment:
How about renaming this to CreateSalt and GetSalt, code will be more readable

Reply:
As mentioned above, if you look at the schema it's like key value pair, which can be used to store different types od data so to generalise it, I have given such naming.